Competition/src/main/java/competition/subsystems/elevator/ElevatorSubsystem.java

```diff
@@ -60,20 +67,30 @@ public ElevatorSubsystem(CommonLibFactory factory, PropertyFactory propManager,
         }
         elevatorStandardPower = propManager.createPersistentProperty("StandardPower", 1);
         distanceBetweenLevels = propManager.createPersistentProperty("DistanceBetweenLevels", 1);
-        brakePowerLimit = propManager.createPersistentProperty("BrakePowerLimit", .05);
-        currentCalibrationSensorPosition = propManager
-                .createPersistentProperty("CalibrationSensorPosition", -1);
+        brakePowerLimit = propManager.createPersistentProperty("BrakePowerLimit", 0.05);
+        currentCalibrationSensorPosition = propManager.createPersistentProperty("CalibrationSensorPosition", -1);
+        armDeadBand = propManager.createPersistentProperty("Arm Deadband", 0.05);
+        timeUntilRaising = propManager.createPersistentProperty("Time Until Arm Raises", .02);
+        positivePowerLatch = new Latch(false, EdgeType.Both, edge -> {
+            if(edge == EdgeType.RisingEdge) {
+                log.info("Arm wants to rise");
+                intialTime = XTimer.getFPGATimestamp();
```

Comment:
might be good to rename `initialTime` to something more descriptive, like `raiseArmRequestedTime`